deploy/config/config_dev.toml

```diff
@@ -0,0 +1,45 @@
+ticket_server_cors_allow_headers = "All"
+ticket_server_cors_allow_origins = ["https://data.umccr.org", "https://data.dev.umccr.org"]
+ticket_server_cors_allow_methods = "All"
+ticket_server_cors_allow_credentials = true
+ticket_server_cors_max_age = 300
+
+data_server_enabled = false
+
+name = "umccr-htsget-rs"
+version = "0.1"
+organization_name = "UMCCR"
+organization_url = "https://umccr.org/"
+contact_url = "https://umccr.org/"
+documentation_url = "https://github.com/umccr/htsget-rs"
+environment = "dev"
+
+[[resolvers]]
```

Comment:
Interesting config arrangement... I'd have expected it not to be so many `[[resolvers]]` sections but a single section collating all of the buckets in this case, but this is just a nitpick, don't worry ;)

Reply:
I think this is just down to how a TOML [array of tables](https://toml.io/en/v1.0.0#array-of-tables) is structured. The alternative is to use a more JSON-like syntax and structure them as inline tables. It is also very straightforward to change config file formats, maybe we could see how something like YAML looks later?